paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -741,6 +741,17 @@
     data_type : dtype
   interfaces : paddle::dialect::InferSymbolicShapeInterface
 
+- op : fused_adam
```

```suggestion
- op : fused_adam_
```
这里我们要和动态图算子保持一致，另外这个算子只有inplace版本，对PIR，所有下划线结尾的都是inplace算子